accelerator/roles/intel/vars/main.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2024 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+intel_gaudi_device_pattern: "Processing accelerators: Habana Labs Ltd."
+
+intel_habana_packages:
+  - habanalabs-container-runtime
```

Comment:
container-runtime should be part of scheduler.yml. just after kubernetes installation we can call k8s_habana_container_runtime role

Reply:
Will move to scheduler